Formula/orocos-kdl.rb

```diff
@@ -0,0 +1,33 @@
+class OrocosKdl < Formula
+  desc "Orocos Kinematics and Dynamics C++ library"
+  homepage "http://www.orocos.org/kdl"
+  url "https://github.com/orocos/orocos_kinematics_dynamics/archive/v1.3.1.tar.gz"
+  sha256 "aff361d2b4e2c6d30ae959308a124022eeef5dc5bea2ce779900f9b36b0537bd"
+
+  depends_on "cmake" => :build
+  depends_on "eigen@3.2"
+
+  def install
+    cd "orocos_kdl" do
+      system "cmake", ".", "-DEIGEN3_INCLUDE_DIR=#{Formula["eigen@3.2"].opt_include}/eigen3",
```

Comment:
still references 3.2

Reply:
I must have borked the `git commit --amend`.

Reply:
Fixed